### PR TITLE
#460 - Added the Image Info annotation to the Peregrine Sling Module

### PR DESCRIPTION
**Pull request by schaefa**
Fix for issue #460

### CODE REVIEW

File: platform/base/core/pom.xml [hunk 1]
```diff
@@ -58,7 +58,9 @@
                             com.peregrine.replication,
                             com.peregrine.transform,
                             com.peregrine.transform.*,
-                            com.peregrine.commons.util
+                            com.peregrine.commons.util,
+                            com.peregrine.model.api,
+                            com.peregrine.model.impl
```
Comment (reusr1) [inline platform/base/core/pom.xml:63]: do we really need to export the impl here?

File: platform/base/core/pom.xml [hunk 2]
```diff
@@ -248,6 +242,10 @@
             <groupId>javax.inject</groupId>
             <artifactId>javax.inject</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains</groupId>
```
Comment (reusr1) [inline platform/base/core/pom.xml:246]: do we need to rely on org.jetbrains for this? isnt @NotNull an existing annotation in java?

**Review by reusr1 — CHANGES_REQUESTED** [review]
@schaefa can we remove the use of org.jetbrains here for the @NotNull annotation?

File: platform/base/core/pom.xml (hunk 1 above)
Comment (reggie7) [reply platform/base/core/pom.xml:63]: #671 handles that

File: platform/base/core/pom.xml (hunk 2 above)
Comment (reggie7) [reply platform/base/core/pom.xml:246]: #671 handles that

**Comment by reggie7** [comment]
Replaced by #671

**Pull request closed without merging**